### ISSUE TITLE
testAnalyzeAndDeleteSpecificAnalysisWithSubset fails unpredictably

### ISSUE THREAD

**Issue by arteymix** [PavlidisLab/Gemma#635]
```
java.lang.AssertionError: We should not be keeping non-number pvalues (null or NaNs)
	at ubic.gemma.core.analysis.expression.diff.DifferentialExpressionAnalyzerServiceTest.testAnalyzeAndDeleteSpecificAnalysisWithSubset(DifferentialExpressionAnalyzerServiceTest.java:152)
```

**Comment by arteymix** [comment]
This was caused by a left-over fixture from another test. It's been addressed in f58bcecca2fc1db49bcd59b8efa3ec3e0dc1a873.

**Comment by arteymix** [comment]
Duplicates of #419 .
